deebot_client/util.py

```diff
@@ -18,6 +24,28 @@ def md5(text: str) -> str:
     return hashlib.md5(bytes(str(text), "utf8")).hexdigest()  # noqa: S324
 
 
+def decompress_7z_base64_data(data: str) -> bytes:
+    """Decomporess base64 decoded 7z compressed string."""
+    _LOGGER.debug("[decompress7zBase64Data] Begin")
```

Comment:
```suggestion
```

A old left over and this log entry is a little bit useless

Reply:
removed